docs/Project-Configuration/Project-Commands.md

```diff
@@ -352,24 +352,26 @@ by a patch submission.
 
 Parameters:
 
--   `dir`: the directory to clone into
--   `revisions`: For commit builds, each module should be passed as
-    `<module_name> : ${<module_name>_rev}`. For patch builds, the hash
+- `dir`: the directory to clone into
+- `revisions`: For commit builds, each module should be passed as
+    `<module_name> : ${<module_name>_rev}` (these are loaded from the [manifest](../API/REST-V2-Usage.md#manifest) 
+    at the beginning of the command). 
+    For patch builds, the hash
     must be passed directly as `<module_name> : <hash>`. Note that this
     means that for patch builds, editing the
     ["modules"](Project-Configuration-Files.md#modules)
     section of the project config will not change the checked out hash.
--   `token`: Use a token to clone instead of the ssh key on the host.
+- `token`: Use a token to clone instead of the ssh key on the host.
     Since this is a secret, it should be provided as a project
     expansion. For example, you could provide an expansion called
     "github_token" and then set this field to \${github_token}.
     Evergreen will populate the expansion when it parses the project
     yaml.
--   `clone_depth`: Clone with `git clone --depth <clone_depth>`. For
+- `clone_depth`: Clone with `git clone --depth <clone_depth>`. For
     patch builds, Evergreen will `git fetch --unshallow` if the base
     commit is older than `<clone_depth>` commits.
--   `shallow_clone`: Sets `clone_depth` to 100.
--   `recurse_submodules`: automatically initialize and update each
+- `shallow_clone`: Sets `clone_depth` to 100.
```

Comment:
I know you didn't change this doc, but while we're at it, should we mention that implies you can't set both `clone_depth` and `shallow_clone` simultaneously?

Reply:
Sure thing